tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+	"compilerOptions": {
+		"module": "commonjs",
+		"target": "es5",
+		"noImplicitAny": false,
```

Comment:
Suggest we make this true.

Reply:
Can't as we do have any in our code, plus, it means all dependencies will also need to follow this rule, which is problematic.  Nothing wrong with using any, just don't want to use it abundantly.